packages/api-client/src/deadLetters.ts

```diff
@@ -0,0 +1,36 @@
+import { DeadLetterArchivePayload } from '@cumulus/types/api/dead_letters';
+import { invokeApi } from './cumulusApiClient';
+import { ApiGatewayLambdaHttpProxyResponse, InvokeApiFunction } from './types';
+
+/**
+ * POST a request to start a migrationCounts run
+ *
+ * @param {Object} params              - params
+ * @param {string} params.prefix       - the prefix configured for the stack
+ * @param {Object} params.payload      - payload to post to the endpoint
+ * @param {Function} params.callback   - async function to invoke the api lambda
+ *                                       that takes a prefix / user payload.  Defaults
+ *                                       to cumulusApiClient.invokeApi
+ * @returns {Promise<Object>}          - promise that resolves to the output
+ *                                       of the API lambda
+ */
+export const postDeadLetters = async (params: {
```

Comment:
Technically this should have a unit.   We could certainly have a discussion about the unit tests in this package....

Reply:
Fair point. I'm also interested in a naming debate on this endpoint.

Reply:
https://github.com/nasa/cumulus/pull/2181/commits/4b6a66a596cc05c31fd86bfa9f52cb254dd6ab14